distributions/build.gradle

```diff
@@ -73,4 +73,4 @@ gradle.projectsEvaluated {
 }
 
 // Building the shadow (fat) jar after compiling sources.
-tasks.assemble.doLast { tasks.shadowJar.execute() }
+assemble.dependsOn(shadowJar)
```

Comment:
It's a good practice to have end of line character. I would recommend adding it here.